src/main/java/webserver/Router.java

```diff
@@ -36,21 +45,22 @@ public String route(Request request) {
         assert routes != null;
         Optional<Callback<Request, String>> controller = routes.entrySet()
                                                   .stream()
-                                                  .filter( entry -> routeMatcher(requestPath, entry.getKey()))
+                                                  .filter(entry -> routeMatcher(requestPath, entry.getKey()))
```

Comment:
:tada: Streaming!  Old school Java programmers lose their minds when they see this, even though Java 8 has been out for over 5 years now.  Now that you've done Clojure, you may proceed fearless into the use of the more functional style.

Reply:
This was fun to look up! I saw it on a blogpost and took a bit to walk through but I really liked it.